app/models/like.rb

```diff
@@ -0,0 +1,11 @@
+class Like < ApplicationRecord
+  belongs_to :post
+  after_create :update_post_likes_counter
```

Comment:
- [ OPTIONAL ] Great job making use of one of Rails' active call-back methods. However, I would like to suggest that I will suggest that you consider using another one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) : `after_save` to call the `#update_user_posts_counter` method every time a new post has been saved to database, then make the `update_user_posts_counter` method private. 

- Example Implementation
```ruby
  # app/models/post.rb
  after_save :update_post_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end

```